src/import-mongo.js

```diff
@@ -0,0 +1,74 @@
+import Promise from 'bluebird';
+import CouchBulkImporter from 'couchbulkimporter';
+import mongo from 'mongodb';
+import BulkBadger from 'bulkbadger';
+import { getUrlFromCluster } from './utils';
+
+export function cli (cluster, database,  mongourl, collection) {
+  if (!cluster || !database || !mongourl || !collection) {
+    const msg = [
+      'Usage:',
+      '',
+      'nmo import-mongo <MongoDB-url> <collection> <clustername> <database>',
+      'nmo import-mongo <MongoDB-url> <collection> <url> <database>'
+    ].join('\n');
+    const err = new Error(msg);
+    err.type = 'EUSAGE';
+
+    throw err;
+  }
+
+  return importmongo(cluster, database, mongourl, collection);
+}
+
+export function validateMongoUrl (url) {
+  return /mongodb:\/\//.test(url);
+}
+
+
+export default importmongo;
+function importmongo (mongourl, collection, cluster, database) {
+  return new Promise((resolve, reject) => {
+    if (!validateMongoUrl(mongourl)) {
+      const err = new Error('Invalid MongoDB url');
+      reject(err);
+      return;
+    }
+
+    mongo.connect(mongourl, function (err, db) {
+      if (err) {
+        reject(err);
+        return;
+      }
+
+      const clusterUrl = getUrlFromCluster(cluster);
+      const col = db.collection(collection);
+      col.find({}, {})
+        .on('error', function (err) {
+          err.message = 'Error fetching collection - ' + err.message;
+          reject(err);
+        })
+        .pipe(new BulkBadger())
+        .on('error', function (err) {
+          err.message = 'Error - ' + err.message;
+          if (/CouchDB server answered/.test(err.message)) {
+            err.type = 'EUSAGE';
+          }
+          reject(err);
+        })
+        .pipe(new CouchBulkImporter({
+          url: clusterUrl + '/' + database
+        }))
+        .on('error', function (err) {
+          err.message = 'Error migration incomplete - ' + err.message;
+          reject(err);
+        })
+        .on('finish', function () {
+          db.close();
+          console.log('Migration complete!');
+          resolve();
```

Comment:
for not existing mongo databases it says:

```
(12:44:34) [robert@tequila-work] ~/apache/couchdb-nmo (pr/16) $ ./bin/nmo-cli.js import-mongo mongodb://127.0.0.1:27017 restaurants12 anemone blerg
Migration complete!
```

but couchdb does not have any database created. this puzzles me. should we say to the user that we did not do any work?